examples/postgres/package.json

```diff
@@ -35,11 +35,11 @@
     "@opentelemetry/exporter-zipkin": "^0.19.0",
     "@opentelemetry/instrumentation": "^0.19.0",
     "@opentelemetry/instrumentation-http": "^0.19.0",
-    "@opentelemetry/instrumentation-pg": "^0.15.0",
+    "@opentelemetry/instrumentation-pg": "file:///Users/neumanns/Projects/opentelemetry-js-contrib/plugins/node/opentelemetry-instrumentation-pg",
```

Comment:
Please undo this

Reply:
🤦‍♂️ ... good catch. I changed that back.